packages/core/tocco-ui/src/EditableValue/typeEditors/DateTimeEdit.js

```diff
@@ -6,9 +6,9 @@ import {injectIntl} from 'react-intl'
 import {atMostOne, momentJStoToFlatpickrFormat} from '../utils'
 import LazyDatePicker from './LazyDatePicker'
 
-export const DateTimeEdit = props => {
-  const momentDateFormat = moment().locale(props.intl.locale)._locale.longDateFormat('L')
-  const momentTimeFormat = moment().locale(props.intl.locale)._locale.longDateFormat('LT')
+export const DateTimeEdit = ({intl, options, onChange, value, immutable, events, placeholder}) => {
+  const momentDateFormat = moment().locale(intl.locale)._locale.longDateFormat('L')
```

Comment:
Can we call `moment().locale(intl.locale)._locale.` only once inside a seperate var and then extract the desired date format?